docs/branding/branding.mdx

```diff
@@ -0,0 +1,82 @@
+<div style={{
+  display: 'flex',
+  flexDirection: 'row',
+  gap: '40px',
+  justifyContent: 'center'  
+}}>
+  <div style={{
+    background: '#539E8A', 
+    color: 'white', 
+    width: 200, 
+    height: 200, 
+    display: 'flex', 
+    justifyContent: 'center', 
+    alignItems: 'center',
+    borderRadius: 20,
+    fontSize: 20
+  }}>
+  <center>
+  <b>Babble Sea Green</b><br />#539E8A
+  </center>
+  </div>
+  <div style={{
+    background: '#F4C2C2', 
+    color: 'black', 
+    width: 200, 
+    height: 200, 
+    display: 'flex', 
+    justifyContent: 'center', 
+    alignItems: 'center',
+    borderRadius: 20,
+    fontSize: 20
+  }}>
+  <center>
+  <b>Babble Pink</b><br />#F4C2C2
+  </center>
+  </div>
+</div>
+
+## Guidelines
+
+We encourage innovation and collaboration, but please adhere to the following guidelines when using our intellectual property:
+
+### **Logo Usage**
+- **Do not modify the logo** - Never stretch, distort, rotate, or alter the proportions of the logo
+- **Maintain clear space** - Always leave adequate space around the logo
+- **Ensure visibility** - Maintain sufficient contrast with backgrounds
+- **No overlays** - Do not place other graphics, text, or elements on top of the logo
+- **Use provided versions** - Only use official logo files from our asset pack
+- **Preffered Font** - IBM Plex Sans italics + bold
+
+### **Brand Attribution**
+- **Required attribution** - When using Project Babble technology, please provide appropriate attribution to "Project Babble" with a link to our official repository
+- **Naming conventions** - Products built using our technology should use phrasing like "**Powered by Project Babble**" or "**Built with Project Babble**" rather than incorporating "Babble" directly in the product name
+
+### **Legal Requirements**
+- **Source First License** - All usage must comply with our source first license terms available [here](https://raw.githubusercontent.com/Project-Babble/ProjectBabble/refs/heads/main/LICENSE.md).
+- **Commercial Applications** - We welcome commercial use of our technology through our partnership program. While our license permits commercial applications, becoming a Project Babble partner ensures compliance with all terms and unlocks additional support. Partnering with us is straightforward—we're eager to collaborate with businesses of all sizes. To discuss partnership opportunities and begin integrating our technology into your commercial products, please contact our team at partners@projectbabble.com. We look forward to supporting your commercial endeavors!
+
+### **Color Usage**
+- **Primary palette** - Stick to our official colors when representing the brand
+- **Contrast requirements** - Ensure text remains legible against colored backgrounds
+- **Color combinations** - Follow accessibility standards when combining our brand colors
+
+For complete branding guidelines or questions about specific use cases, please contact our team at [contact email to be added].

```

Comment:
Add the email here as well: [projectbabblevr@gmail.com](mailto:projectbabblevr@gmail.com)